modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/view_stack_trace.jsp

```diff
@@ -0,0 +1,20 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/publications/init.jsp" %>
+
+<%
+long backgroundTaskId = GetterUtil.getLong(renderRequest.getParameter("backgroundTaskId"));
+
+BackgroundTask backgroundTask = BackgroundTaskLocalServiceUtil.getBackgroundTask(backgroundTaskId);
+%>
+
+<clay:container-fluid>
+	<div>
+		<liferay-ui:message key="<%= backgroundTask.getStatusMessage() %>" />
+	</div>
+</clay:container-fluid>
```

Comment:
Can you share a screenshot for what this looks like in the UI?

Reply:
This is what it looks on the UI with the changes  ![Pasted image](https://github.com/liferay-publications/liferay-portal/assets/48836305/a622e5af-0355-4e87-b7a4-cbb16aca5493)

Reply:
Let's wrap the stack trace container in a sheet by adding these classes to the div `<div class="sheet sheet-lg">`. We will also need a heading to indicate what the stack trace is for. Maybe add this to the header: `P1 failed to publish.`

Reply:
After these changes: ![image](https://github.com/liferay-publications/liferay-portal/assets/48836305/4f2d8b17-c4b2-46ce-8427-03c61cd2bd25)